guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -38,6 +38,11 @@ For more information, see {AdministeringDocURL}Backing_Up_Server_and_Proxy_admin
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 For more information about changes in the API, see the Red Hat Knowledgebase article https://access.redhat.com/articles/4396911[API Changes Between {Project} Versions] on the Red{nbsp}Hat Customer Portal.
 
+ifdef::foreman-el,katello,satellite[]
+NOTE: {Project} {ProjectVersion} is planned to be released only on {EL} 8. You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
```

Comment:
*planned*?
Foreman/Katello already only support EL8 since Foreman 3.4 (Katello 4.6).
For Satelite, the docs won't be published before the Product isn't released, so also not "in the future"